src/Tribe/Main.php

```diff
@@ -4772,12 +4596,152 @@ public function maybe_set_currency_position_with_post( $reverse_position, $post_
 			return $reverse_position;
 		}
 
+		/**
+		 * Filters the chunkable post types.
+		 *
+		 * @param array $post_types
+		 * @return array The filtered post types
+		 */
+		public function filter_meta_chunker_post_types( array $post_types ) {
+			$post_types = array_merge(
+				$post_types,
+				array(
+					self::POSTTYPE,
+					self::VENUE_POST_TYPE,
```

Comment:
`Tribe__Events__Venue::POSTTYPE` and `Tribe__Events__Organizer::POSTTYPE`

Reply:
Good catch.